src/types/bundle/serialized.ts

```diff
@@ -23,24 +23,24 @@ type SerializedVerificationData = {
       keyId: string;
     };
     kindVersion:
-      | {
-          kind: string;
-          version: string;
-        }
-      | undefined;
+    | {
```

Comment:
hm not sure what happened here, I think `prettier` just ran. Should this be set in config somewhere?